communitiesuk/funding-service-design-application-store
# FS-2501 Automate import : simple submit message

https://dluhcdigital.atlassian.net/browse/FS-2501

### Description
This PR is a part of the work for FS-2501

- Implemented SQS Submit message function & endpoint.
- Updated config for the SQS.
- Added unit tests for SQS
- Added & updated readme.

linked PR https://github.com/communitiesuk/funding-service-design-assessment-store/pull/271

### How to test
- Pull the changes locally and also from [assessment-store](https://github.com/communitiesuk/funding-service-design-assessment-store/pull/271) and docker-runner
- rebuild the docker containers.
- Go to frontend & submit an application
- Observe that application is automatically imported to assessment in 60 seconds time

### Screenshots of UI changes (if applicable)
